modules/special-rules/pages/super-heavy-weapons.adoc

```diff
@@ -7,48 +8,47 @@ These rules define the effects of super-heavy weapons; see the specific unit dat
 
 == Shooting with super-heavy weapons
 
-* Except where we state otherwise, you may shoot super-heavy weapons at any target within range.
-That is, you can pick out individual targets within a detachment.
-
-* Where a unit that is equipped with super-heavy weapons is on Assault orders, and you wish it to shoot in the Shooting phase: For each super-heavy weapon you wish to shoot, you must first roll a 4 or more on a D6.
-This is because it is much harder to target the enemy and route enough power to a super-heavy weapon while the unit hastens to engage the enemy more closely.
+Half Firepower and 'lock on'::
+Where a game effect forces you to halve the Firepower of a detachment or unit, you also suffer a reduction in the effectiveness of its super-heavy weaponsfootnote:[Common game effects that force you to halve Firepower include Assault orders and some particular critical damage effects for War Engine units.].
+In these situations, you must test to *lock on* with each super-heavy weapon before you can shoot with it.
 +
-+[TODO: Consider how this should work if we're using standard rules in which war engines don't actually need specific orders.]+
+.Lock on
+For each super-heavy weapon that you wish to shoot, roll a D6.
+On a roll of 4+ the weapon locks on successfully, so you may shoot with it.
+Otherwise, you may not shoot with it this time.
 
-* Where a unit that is equipped with super-heavy weapons is on Overwatch orders: You may re-roll misses when you shoot with these weapons, just as you do with regular weapons.
+Overwatch::
+Where a unit that is equipped with super-heavy weapons is on Overwatch orders you may re-roll misses when you shoot with these weapons, just as you do with dice that you get from Firepower.
 
 == Super-heavy weapons in firefights
 
-* Where a unit that is equipped with super-heavy weapons is in a firefight, and a super-heavy weapon doesn't have a basic Firepower value: Treat that weapon as if it has a Firepower value of 1.
+Where a super-heavy weapon doesn't have a basic Firepower value, count it as having a Firepower value of 1 for a firefight.
```

Comment:
```suggestion
Where a super-heavy weapon doesn't have a basic Firepower value, count it as having a Firepower value of 1 for a firefight.
```

shooting-with-war-engines.adoc will need to be adjusted to match this.

Reply:
You already covered this point in that topic I think:

>In addition, a weapon mount that does not have a plain Firepower value, such as a {death-ray} or {heavy-barrage} weapon, counts as a single super-heavy weapon, so:
>
>* is suppressed by 1 Blast marker
>* is one weapon for placing Blast markers
>* rolls once to lock on if Firepower is halved
>* contributes 1 Firepower in a firefight

But I'll see if I can unify the language at all when I edit it. Or did you mean something else?

Also, the statement above blurs the line between weapon mount and weapon -- a weapon mount doesn't have Firepower, only a weapon in that mount has Firepower...? Although perhaps it comes about because of things like sponsons on a Shadowsword, which are not defined as a weapon in their own right, only as a weapon mount. Hmmm. In that case perhaps it is simplest to accept a little blending like this, where sometimes we treat 'mount' as a proxy for (or super-set of) 'weapon'? I'm happy to leave that detail alone for now, we can always revisit. Just thought I'd plant the seed in your mind in case it overlaps with anything else in future.

Reply:
> You already covered this point in that topic I think:
> 
> > * contributes 1 Firepower in a firefight
> 
> But I'll see if I can unify the language at all when I edit it. Or did you mean something else?

Yes, that's what I was referring to. They both used the same 'contributes 1 Firepower in a firefight' phrasing. Without wanting to step on your toes as TW, I'll always try to present you with consistent phrasing because it's such an important contributor to rules comprehension.

For what it's worth, I went with 'contributes X' as I've had bad experiences with 'count as having X' or 'treat as having X' leading to messy rules interactions - where an item counts as different 'types' of item in different situations, any rule that depends on 'type' can cause issues. That said, I can't think of any rules issues for this one.

While I think of it, 'plain Firepower value' in shooting-with-war-engines.adoc should probably be 'basic Firepower value'


> Also, the statement above blurs the line between weapon mount and weapon -- a weapon mount doesn't have Firepower, only a weapon in that mount has Firepower...? Although perhaps it comes about because of things like sponsons on a Shadowsword, which are not defined as a weapon in their own right, only as a weapon mount. Hmmm. In that case perhaps it is simplest to accept a little blending like this, where sometimes we treat 'mount' as a proxy for (or super-set of) 'weapon'? I'm happy to leave that detail alone for now, we can always revisit. Just thought I'd plant the seed in your mind in case it overlaps with anything else in future.

Yes, exactly. 'Weapon mount' has to cover a weapon that consists of one SHW, or multiple SHWs, or just a Firepower value.

If we can come up with clearer language at some point it would be good, but I fear the text would get extremely convoluted.

Reply:
>For what it's worth, I went with 'contributes X' as I've had bad experiences with 'count as having X' or 'treat as having X' leading to messy rules interactions 

Now I think about it, I do have a slight sense of unease with the likes of 'count as having X', but haven't quite identified exactly why. I'll look at that other bit of text again and re-apply your 'contributes X' construction.

>While I think of it, 'plain Firepower value' in shooting-with-war-engines.adoc should probably be 'basic Firepower value'

Plain/basic/base/regular/normal/inherent... there are lots of possibilities here I think. I'm fine with basic though, so I'll go with that if you prefer it. I think there might be other synonyms elsewhere, which can be a bad thing with special terms — but I don't think this is so much of a special term as a general sense that people will understand anyway. We can always hunt down synonyms and replace with a single term later if we do uncover a problem.

Reply:
That said, it does get hard to avoid. Even with that bullet where you have 'contributes', the lead-in sentence has 'counts as':

>In addition, a weapon mount that does not have a plain Firepower value, such as a {death-ray} or {heavy-barrage} weapon, counts as a single super-heavy weapon, so:

It might not always be worth the worry, because it can make sentences pretty convoluted/long and I'm not sure that is any better.

Reply:
Plain/basic/base/regular/normal/inherent - that was just another consistency one. If 'basic' is used here, then it should be 'basic' in shooting-with-war-engines.adoc.

I have no strong preference on which word is chosen, although GW used 'basic'.

Reply:
I restructured it to single-source via a partial, and used 'contribute' in that partial. See what you think. I'll resolve this conversation, as it is now 'outdated'.